bin/benchpark_tags.yaml

```diff
@@ -0,0 +1,85 @@
+benchpark-tags:
+  application-domain:
+    asc: null # Advanced Simulation and Computing
+    astrophysics: null
```

Comment:
Why format this like a dictionary vs.

```
benchpark-tags:
  application-domain:
  -  asc: # Advanced Simulation and Computing
  - astrophysics: null
```

one reason I could see is to add properties like a description, e.g.:

```
benchpark-tags:
  application-domain:
    asc: 
      description: "advanced simulation and computing"
      group: "" # maybe this is like a suite?
```

that way we could say something like `benchpark explain-tag "asc"`. I'm not sure if this should live in benchpark though, since the tags are managed independently (i.e. in ramble, which can change whenever it wants).